Orange/widgets/visualize/owsilhouetteplot.py

```diff
@@ -340,6 +346,15 @@ def _ensure_matrix(self):
             else:
                 assert False, "invalid state"
 
+    def _update_avg_silhouette(self):
+        # update the average silhouette score
+        if self._silhouette is not None and len(self._silhouette) > 0:
+            avg_score = np.mean(self._silhouette)
+            self.avg_silhouette_label.setText(
+                f"<b>Silhouette:</b> {avg_score:.4f}")
+        else:
+            self.avg_silhouette_label.setText("<b>Silhouette:</b> N/A")
```

Comment:
Must it be bold? I don't think we use bold anywhere else (or at least not often).

Reply:
We used bold in Data Info widget, I copied the style from there. But I agree it sticks out, and will correct. Let me work on this in the following days.